src/explore-education-statistics-common/src/modules/release/components/__tests__/ReleaseDataAndFiles.test.tsx

```diff
@@ -0,0 +1,85 @@
+import ReleaseDataAndFiles from '@common/modules/release/components/ReleaseDataAndFiles';
+import { Release } from '@common/services/publicationService';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { FileInfo } from '@common/services/types/file';
+
+describe('ReleaseDataAndFiles', () => {
+  const testRelease = {
+    id: 'r1',
+    downloadFiles: [
+      {
+        id: 'f1',
+        name: 'All files',
+        type: 'Ancillary',
+      },
+      {
+        id: 'f2',
+        name: 'data file',
+        type: 'Data',
+      },
+      {
+        id: 'f3',
+        name: 'ancillary file',
+        type: 'Ancillary',
+      },
+    ] as FileInfo[],
+    hasMetaGuidance: true,
+    hasPreReleaseAccessList: true,
+    publication: {
+      slug: 'p-slug',
+    },
+    slug: 'r-slug',
+  };
+
+  test('renders the data and files accordion', () => {
+    render(
+      <ReleaseDataAndFiles
+        release={testRelease as Release}
+        renderDownloadLink={file => <a href="/">{file.name}</a>}
+        renderMetaGuidanceLink={<span>mock meta guidance link</span>}
```

Comment:
Let's use actual anchor elements for both of these to be consistent and make the test assertion stricter (see below).